Script/char.dat

```diff
@@ -802,6 +840,61 @@ guard
     }
   }
+
+  Config EMISSARY;
+  {
+    AttributeBonus = 30;
+    DefaultIntelligence = 20;
+    DefaultWisdom = 20;
+    EyeColor = rgb16(0, 153, 255);
+    SkinColor = rgb16(230, 140, 100);
+    ClothColor = rgb16(128, 128, 128);
+    HeadBitmapPos = 96, 368;
+    Helmet = 0;
+    BodyArmor = STEEL bodyarmor(PLATE_MAIL) { Enchantment = 2; }
+    RightWielded = OCTIRON OCTIRON meleeweapon(BASTARD_SWORD) { Enchantment = 2; }
+    LeftWielded = EBONY_WOOD shield { Enchantment = 2; }
+    RightGauntlet = STEEL gauntlet { Enchantment = 2; }
+    RightBoot = STEEL boot { Enchantment = 2; }
+    DefaultName = "sir Lancelyn";
+    NameSingular = "emissary";
+    PostFix = "of Aslona";
+    CWeaponSkillHits = { 2, 500, 500; }
+    RightSWeaponSkillHits = 200;
+    LeftSWeaponSkillHits = 200;
+    PanicLevel = 50;
+    CanBeGenerated = false;
+    IsNameable = false;
+    IsUnique = true;
+    CanBeCloned = false;
+    IsPolymorphable = false;
+    UndeadVersions = false;
+    TamingDifficulty = 25;
+    AttachedGod = CRUENTUS;
+    FriendlyReplies =
+    {
+      12,
+      "\"I shouldn't be saying this so openly, but Aslona is in dire straits and *needs* any help it can get.\"",
+      "\"I've heard other emissaries were sent from Aslona even to goblin lands to negotiate assistance.\"",
+      "\"After the old king was assassinated and the Rebellion started, Aslona lost much of its former power.\"",
+      "\"I will see the Rebellion crushed and every last rebel executed for their crimes!\"",
+      "\"My king is young and inexperienced, his army only barely holds the rebels back. My only hope is I can return to him successful.\"",
+      "\"Bandits are roving free through the forests of Aslona and you can never be sure which town supports the king and which the Rebellion. What happened to my country?\"",
+      "\"My father was a trusted general of the old king, and the first one to meet with the rebel army on a battlefield after the regicide. Alas, he was betrayed by his slave-soldiers and murdered along with all of his loyal knights.\"",
+      "\"I see a representative of the orcish freeholds arrived. Don't trust him! Those orcish vultures would love to feast on the carcass of Aslona, but I won't let that happen.\"",
+      "\"I hope the negotiations over in goblin lands go well. As I heard, my king offers to free all goblin slaves in Aslona in exchange for military assistance against the orcish freeholds.\"",
+      "\"I don't understand why does the high priest tolerate that filthy orc here! No orc can be trusted, they are all raiders and murderers with no honor.\"",
+      "\"The orcish freeholds have set their sights on aslonian wealth long ago and were a bother on the borders since then. Right now, I don't know whether we are strong enough to fend them off by ourselves.\"",
+      "\"Hm, that elven ambassador, those perky... eyes of hers. As if she could see right through you, right?\"";
```

Comment:
This line being funny is subjective opinion. ;) But I will remove it, because it is not very consistent with the rest of his lines.